code/modules/mob/mob_grab.dm

```diff
@@ -360,7 +360,7 @@
 						assailant.attack_log += text("\[[time_stamp()]\] <font color='red'>Pressed fingers into the eyes of [affecting.name] ([affecting.ckey])</font>")
 						affecting.attack_log += text("\[[time_stamp()]\] <font color='orange'>Had fingers pressed into their eyes by [assailant.name] ([assailant.ckey])</font>")
 						msg_admin_attack("[key_name(assailant)] has pressed his fingers into [key_name(affecting)]'s eyes.")
-						var/obj/item/organ/eyes/eyes = affected.internal_organs_by_name["eyes"]
+						var/obj/item/organ/internal/eyes/eyes = affected.internal_organs_by_name["eyes"]

```

Comment:
doesn't matter, will never be used on this server, but if you want to, would appreciate